docs/targets/googlesheets.md

```diff
@@ -71,7 +71,7 @@ ko apply -f config/
 kubectl apply -f samples/
 ```
 
-3. If everything worked sucessfully a new namespace "triggermesh-knative-targets" should be now be created with a Googlesheets adapter and controller pod active and ready!
+3. If everything worked successfully a new namespace "triggermesh-knative-targets" should be now be created with a Google Sheets adapter and controller pod active and ready!
```

Comment:
Google Sheets should be GoogleSheets.

Reply:
@cab105  Thanks for the review. I do not want to be difficult, but when I searched for GoogleSheets in Google, they do have it as two separate words. I am happy to make this change, I just was not sure if there had already been a discussion about the spelling :) 
<img width="200" alt="Screen Shot 2020-11-12 at 9 57 05 AM" src="https://user-images.githubusercontent.com/54158919/98956018-8f89cb00-24cd-11eb-8bfd-349602f5ed86.png">

Reply:
Too late. :-P In other cases I do see it references as just Sheets. Feel free to space it out.

Reply:
haha thanks